runtime-modules/storage/src/tests/mod.rs

```diff
@@ -6256,9 +6252,10 @@ fn uploading_objects_with_invalid_cid_length_should_fail() {
         let data_object_state_bloat_bond = 10u64;
         set_data_object_state_bloat_bond_value(data_object_state_bloat_bond);
 
+        const ONE_MB: u64 = 1_048_576;
         let initial_balance = ExistentialDeposit::get() as u64
             + data_object_state_bloat_bond
-            + Storage::data_object_per_mega_byte_fee() * DEFAULT_DATA_OBJECTS_SIZE;
+            + Storage::data_object_per_mega_byte_fee() * DEFAULT_DATA_OBJECTS_SIZE / ONE_MB;
```

Comment:
You need to do `DEFAULT_DATA_OBJECTS_SIZE.div_ceil(ONE_MB)`, but since Rust doesn't have this `div_ceil` you can use the `a.div_ceil(b) = (a + b - 1) / b` rule (https://stackoverflow.com/questions/72442853/how-would-you-perform-ceiling-division):
```suggestion
            + Storage::data_object_per_mega_byte_fee() * (DEFAULT_DATA_OBJECTS_SIZE + ONE_MB - 1) / ONE_MB;
```